package.json

```diff
@@ -61,9 +61,6 @@
     "configPath": "tests/dummy/config",
     "after": [
       "ember-cli-fastboot"
-    ],
-    "before": [
-      "broccoli-asset-rev"
```

Comment:
I don't really want to lose this work, although I realize it's still dependent on some unmerged upstream changes. I will push on those again today.

Without `before`, people will get broken production apps with incorrect asset rewriting.

Reply:
Interesting. This caused a circular dep on my machine (ie because [e-cli-fastboot requires broccoli-asset-rev in its `after`](https://github.com/ember-fastboot/ember-cli-fastboot/blob/master/package.json#L78-L80))

Reply:
Right, that's why the version of ember-cli-fastboot in this file is pointing at my fork.

Reply:
![oops](https://media.giphy.com/media/CoDp6NnSmItoY/giphy.gif)